docu/report/aufbau.tex

```diff
@@ -24,12 +60,36 @@ \subsection{Antenne}
 \begin{figure}
     \centering
     \includegraphics[width=\textwidth]{images/antenne.png}
-    \caption{SKT SL43-01 UHF 43 Antenne}\label{antenne}
+    \quelle{\cite{ANTENNE2021}}
+    \caption{SKT SL43-01 UHF 43 Antenne}\label{fig:antenne}
 \end{figure}
+
 \section{Signal}
+Das hier verwendete Signal beruht auf der LTE Technik. Long Term Evolution oder kurz LTE und war vor allem dadurch erfolgreich, dass eine neue Übertragungstechnik verwendet wurde, mit der das Multipath-Fading-Effekt umgangen wurde. Der Multipath-Fading-Effekt tritt auf, wenn die Länge eines Übertragungsschrittes verkleinert wird, was bei einer Verzögerung des Signals dazu führen kann, dass sich die einzelnen Übertragungsschritte überlappen. Die Technik, die dem gegen wirkt, hat die Bezeichnung Orthogonal Frequenz Division Multiplexing (OFDM) Technik. Hierbei wird ein schneller Datenstrom in viele kleine langsame unterteilt, da so kein Multipath-Fading-Effekt auftritt, nach dem Übertragen wird Datenstrom wieder zusammengeführt. OFDM wird in einem der unter Kapitel noch genauer erläutert werden.
```

Comment:
Die funktionsweise von OFDM und _warum_ es Multipath-resistent ist, kommt hier irgendwie nicht so wirklich rüber. Klar kannst du nicht die Quelle 100% zitieren, aber ein paar Kernpunkte fehlen, wie:

* Multipath tritt auf wenn das Signal an Hindernissen reflektiert wird, bevor es beim Empfänger eingeht.
* Resultat von Multipath ist eine Überlagerung mehrerer zeitlich verschobener Kopien vom Signal im Empfänger.
* Durch die Überlagerung ist der Empfänger nicht mehr in der Lage die Symbole zu dekodieren.
* OFDM umgeht das Problem, indem es die einzelnen Symbole länger macht, als der zu erwartende Zeitversatz der durch Multipath entstehen könnte.

Mehr gibt die Quelle zu dem Thema leider nicht her.

```suggestion
Das betrachtete Signal beruht auf der LTE Technik. Long Term Evolution oder kurz LTE ist und war vor allem dadurch erfolgreich, dass eine neue Übertragungstechnik verwendet wurde, mit der das Multipath-Fading-Effekt umgangen wurde. Der Multipath-Fading-Effekt tritt auf, wenn die Länge eines Übertragungsschrittes verkleinert wird, was bei einer Verzögerung des Signals dazu führen kann, dass sich die einzelnen Übertragungsschritte überlappen. Die Technik, die dem gegen wirkt, hat die Bezeichnung Orthogonal Frequenz Division Multiplexing (OFDM). Hierbei wird ein schneller Datenstrom in viele kleine langsame unterteilt, da so kein Multipath-Fading-Effekt auftritt. Nach dem Übertragen wird der Datenstrom wieder zusammengeführt. OFDM wird in den folgenden Abschnitten noch weiter erläutert.
```